addons/mail/static/src/chatter/web/chatter_patch.js

```diff
@@ -319,6 +321,9 @@ patch(Chatter.prototype, {
 
     onActivityChanged(thread) {
         this.load(thread, [...this.requestList, "messages"]);
+        if (this.props.hasParentReloadOnActivityChanged) {
+            this.reloadParentView();
```

Comment:
Make sure load of message and request list above is not done twice due to the reload of the record and the manual call.
Otherwise the load should be moved in else clause.